iib/workers/config.py

```diff
@@ -16,7 +16,7 @@ class Config(object):
     iib_index_image_output_registry = None
     iib_log_level = 'INFO'
     iib_required_labels = {}
-    iib_skopeo_timeout = '30s'
+    iib_skopeo_timeout = '60s'
```

Comment:
Is there a specific reason to do this? If yes, how about bumping this up to 300s based on the discussion in CLOUDDST-472

Reply:
I did this because skopeo is now used to push the image which can take longer. I think 300s is a valid default. I'll change to use it.

Reply:
Done.